exporter/exporterhelper/internal/bounded_memory_queue.go

```diff
@@ -129,9 +133,9 @@ func (q *boundedMemoryQueue) Produce(item interface{}) bool {
 // and releases the items channel. It blocks until all consumers have stopped.
 func (q *boundedMemoryQueue) Stop() {
 	q.stopped.Store(1) // disable producer
+	close(q.items)
 	close(q.stopCh)
```

Comment:
I think we should just simply remove entirely the `stopCh`, and rely on the line 77 to exit for the worker. That is probably the cleanest.